src/components/Select/Select.test.tsx

```diff
@@ -43,10 +44,24 @@ describe("Select ", () => {
       attachTo: container,
     });
     expect(wrapper.find("select").getDOMNode()).toBe(document.activeElement);
+    document.body.removeChild(container);
   });
 
   it("can take null options", () => {
     const wrapper = shallow(<Select options={null} />);
     expect(wrapper.exists()).toBe(true);
   });
 });
+
+describe("Select RTL", () => {
```

Comment:
Great that we write new tests in RTL already, but a bit of a pity to have 2 separate test suits in same file. But I get that it's not the scope of this to rewrite all the tests.

Maybe we need some RTL sprint to migrate all tests in react components.

Reply:
Yeah it's not ideal. I can rewrite these with enzyme if you'd prefer.

Reply:
No, let's not write new tests in enzyme.